Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -169,11 +206,59 @@ equation
           -52}}, color={0,0,127}));
   annotation (Documentation(info="<html>
 <p>
-This blocks implements a AMIGO PID tuning method and the control process is approximated with a first order delay process.
+This block implements an AMIGO PID tuning method and the control process is approximated with a first-order delay (FOD) process.
+</p>
+<p>
+Specifically, this AMIGO PID tuning method is based on asymmetric relay feedback tests, providing process excitation 
+at the frequency intervals relevant for PID control;
+</p>
+<p>
+Based on the response of the control process to the excitation, the parameters of a FODs, time delay and time constant, are calculated;
+</p>
+<p>
+Finally, the parameters of a PID controller are determined according to the parameters of the FOD.
+</p>
+<p>
+This block consists of a relay controller and a PID controller with input gains.
 </p>
 <p>
 Noted that this block can only support a PI controller or a PID controller.
 </p>
+<h4>Breif guidance</h4>
+<p>
+This block is built upon PIDWithInputGains<a href=\"modelica://Buildings.Controls.OBC.Utilities.PIDWithInputGains\">
+Buildings.Controls.OBC.Utilities.PIDWithInputGains</a> and its connections are the same as those of the latter.
```

Comment:
`This block is built based on the <a href=\"modelica://Buildings.Controls.OBC.Utilities.PIDWithInputGains\">
 Buildings.Controls.OBC.Utilities.PIDWithInputGains</a> with the gain inputs calculated by tuning blocks.`

Reply:
Updated this sentence accordingly.